python/lsst/meas/extensions/convolved/convolved.py

```diff
@@ -93,6 +93,13 @@ class BaseConvolvedFluxConfig(Config):
     maxSincRadius = Field(dtype=float, default=10.0,
                           doc="Largest aperture for which to use the sinc aperture code for Kron (pixels)")
     kronRadiusForFlux = Field(dtype=float, default=2.5, doc="Number of Kron radii for Kron flux")
+    registerForApCorr = Field(dtype=bool, default=True,
+                              doc="Register measurements for aperture correction?\n"
+                                  "The aperture correction registration is done at instantiation time\n"
+                                  "because the column names are derived from the configuration rather than\n"
+                                  "being static. Sometimes you want to turn this off, e.g., when you will\n"
+                                  "use aperture corrections derived from somewhere else through the 'proxy'\n"
+                                  "mechanism.")
```

Comment:
I didn't know what "instantiation time" meant, exactly, when first reading this.  Maybe better to say "when the Plugin is instantiated".